scripts/clean-build.sh

```diff
@@ -36,6 +36,8 @@ apt-get install -qq -y \
   pkg-config \
   protobuf-compiler
 
+apt-get clean
```

Comment:
Well, it's still helpful if someone build it without docker but just bare-metal?